core/script_debugger_remote.cpp

```diff
@@ -231,11 +252,11 @@ void ScriptDebuggerRemote::debug(ScriptLanguage *p_script, bool p_can_continue)
 					}
 				}
 
-				{ //locals
-					packet_peer_stream->put_var(locals.size());
+				{ //globals
```

Comment:
The idea of sending globals is fine, but in GDScript, globals are constants and there are a ton of them. Sending them every time seems like too much.

Reply:
Infact only autoloads will be sent in GDScript.